Team 6 Senior Project/SpecimensForm.cs

```diff
@@ -596,5 +609,33 @@ private void specimensDataGridView_DataError(object sender, DataGridViewDataErro
         {
             e.Cancel = true;
         }
+
+        private void templatesToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            Program.CurrentForm = new TemplatesForm();
+            this.Close();
+        }
+
+        private void summaryToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            Program.CurrentForm = new SpecimensSummaryForm();
+            this.Close();
+        }
+
+        private void toolStripDropDownButton1_Click(object sender, EventArgs e)
```

Comment:
625-634 don't do anything and can be deleted. Be sure to remove the matching references in the code behind.